### PR TITLE
Added overview to intro-to-anchor chapter

### PR DESCRIPTION
**Pull request by shivaji43**
### Problem
Added overview to intro-to-anchor chapter and given a brief.

### Summary of Changes
Gave a brief intro to Anchor and it applications and how it is used.

Fixes #

### CODE REVIEW

**Comment by shivaji43** [comment]
@nickfrosty  Can you please check the PR  asap , the submission date is very close

File: content/courses/onchain-development/intro-to-anchor.md [hunk 1]
```diff
@@ -17,6 +17,54 @@ description: "Create your first Solana onchain program in Anchor."
...
+  automatically, reducing the chances of errors.
+- `Automatic Account Validation`: Anchor validates accounts by checking for:
+- `Type correctness`: Ensures that accounts match the expected data types.
+- `Uniqueness`: Ensures that each account is correctly initialized and doesn't
+  conflict with existing ones.
+- `Simplified Data Handling`: Anchor automatically serializes and deserializes
+  the data passed in transactions, making it easier to manage the inputs and
+  outputs of your program.
+- `Instruction Routing`: Anchor routes incoming instructions to the appropriate
+  function, simplifying the process of executing program logic.
```
Comment (mikemaccana) [inline content/courses/onchain-development/intro-to-anchor.md:40]: These should be indented below their parent.

File: content/courses/onchain-development/intro-to-anchor.md [hunk 2]
```diff
@@ -17,6 +17,54 @@ description: "Create your first Solana onchain program in Anchor."
...
+
+Here’s how Anchor works in practice:
+
+Programs are deployed at addresses In the same way, Solana programs can be
+deployed using their program ID, and Anchor programs can also be deployed with a
+unique public key. This keypair is created during the initialization of an
+Anchor project and is stored in the target/deploy directory.
```
Comment (mikemaccana) [inline content/courses/onchain-development/intro-to-anchor.md:47]: 'Programs are deployed at addresses' needs a full stop.

'In the same way, Solana programs can be deployed using their program ID,' - in the same way as what? I thought the previous sentence referred to Solana programs.

'and Anchor programs can also be deployed with a unique public key' - Anchor programs are also Solana programs.

File: content/courses/onchain-development/intro-to-anchor.md [hunk 3]
```diff
@@ -17,6 +17,54 @@ description: "Create your first Solana onchain program in Anchor."
...
+- When you create a new Anchor program, a keypair is generated via the anchor
+  init command.
+- This keypair is essential for deploying the program and is referenced as the
+  program's ID in multiple places, such as Anchor.toml and your program’s main
+  file (typically lib.rs). Using Anchor The core structure of an Anchor project
+  revolves around:
+
+- `Accounts`: These are user or program-specific accounts that store state and
+  data. Anchor makes it easy to define and validate accounts in your program.
+- `Instructions`: These are the actions users and programs can trigger. Anchor
```
Comment (mikemaccana) [inline content/courses/onchain-development/intro-to-anchor.md:58]: No. See [CONTRIBUTING.md. ](https://github.com/solana-foundation/developer-content/blob/main/CONTRIBUTING.md#prose) re instructions.

**Review by mikemaccana — CHANGES_REQUESTED** [review]
This PR needs significant changes to be merged. Please see the comments above and provide an update if you're still interested in working on this.

**Comment by shivaji43** [comment]
I am still interested to work on this and will try to make the changes according to the comments.

**Pull request closed without merging**